kahuna/public/js/image/service.js

```diff
@@ -19,6 +22,30 @@ imageService.factory('imageService', [function() {
         };
     }
 
+    function usages(image) {
+        function usageTitle(usage) {
+            const referenceType = usage.get("usageType") == "print" ? "indesign" : "frontend";
```

Comment:
I thought we were jslinting to use single quotes everywhere, but didn't look like it failed here?